app/cli/src/main/resources/logback.xml

```diff
@@ -0,0 +1 @@
+<configuration />
```

Comment:
nit: Should we just get rid of this ? Or maybe put a example hin here?

Reply:
This is to disable logging in the cli project. Without this we would get logs from parsing some of the arguments which can be confusing to the user